README.md

```diff
@@ -1,564 +1,126 @@
-
-           python $CONDA_PREFIX/scripts/isisVarInit.py
-
-       Executing this script with no arguments will result in $ISISROOT=$CONDA\_PREFIX and $ISISDATA=$CONDA\_PREFIX/data. The user can specify different directories for `$ISISDATA` using the optional value:
-
-           python $CONDA_PREFIX/scripts/isisVarInit.py --data-dir=[path to data directory]
-
-       Now every time the isis environment is activated, $ISISROOT and $ISISDATA will be set to the values passed to isisVarInit.py.
-       This does not happen retroactively, so re-activate the isis environment with one of the following commands:
-
-           for Anaconda 3.4 and up - conda activate isis
-           prior to Anaconda 3.4 - source activate isis
-
-
-### Installation with Docker
-The ISIS production Dockerfile automates the conda installation process above.
-You can either build the Dockerfile yourself or use the
-[usgsastro/isis](https://hub.docker.com/repository/docker/usgsastro/isis)
-image from DockerHub.
-
-#### To build the Dockerfile
-1. Download [the production Docker file](./docker/production.dockerfile)
-2. Build the Dockerfile
-  ```
-  docker build -t isis -f production.dockerfile .
-  ```
-3. Run the Dockerfile
-  ```
-  docker run -it isis bash
-  ```
-
-#### Run run the prebuilt image
-```
-docker run -it usgsastro/isis bash
-```
-
-#### Usage with the ISIS data area
-Usually you'll want to mount an external directory containing the ISIS data.
-The data is not included in the Docker image.
-
-```
-docker run -v /my/data/dir:/opt/conda/data -v /my/testdata/dir:/opt/conda/testData -it usgsastro/isis bash
-```
-
-Then [download the data](#the-isis-data-area) into /my/data/dir to make it accessible inside your
-container.
-
-### Practical Usage with other conda packages
+For more info, see:
 
-If you don't use conda for anything else on your computer, you can
-skip this section.
+- [Tutorials and Guides](https://astrogeology.usgs.gov/docs/)
+- [FAQ](https://astrogeology.usgs.gov/docs/getting-started/using-isis-first-steps/isis-faq/)
+- [ISIS Application Manuals](https://isis.astrogeology.usgs.gov)
 
-If you use conda to install other packages, you may run into
-difficulties with adding the isis conda package to those environments
-or adding other conda packages to the isis environment you just
-created above.  This is because the isis conda package pins a number
-of requirements that may clash with other packages.
+For Development and Contributing, see:
 
-At this time, we recommend creating the isis environment as detailed
-above, and then not adding any other conda packages to it.  This
-is similar to the best practice usage of not adding any conda
-packages to your 'base' conda environment.
+- [Building ISIS](https://astrogeology.usgs.gov/docs/how-to-guides/isis-developer-guides/developing-isis3-with-cmake/)
+- [ISIS Test Data](https://astrogeology.usgs.gov/docs/how-to-guides/isis-developer-guides/obtaining-maintaining-submitting-test-data/)
```

Comment:
We need something that is a more general "this is how you run ISIS tests" not just how to install isis test data. Those linked docs cover it but maybe name this link "'How to run ISIS tests"

Reply:
"How to run ISIS Tests" is currently under the "Building ISIS" Doc.  [Here's a small PR to link to "ISIS Test Data" from "Building ISIS."](https://github.com/DOI-USGS/asc-public-docs/pull/127)  And with that, I can safely remove the ISIS Test Data link from the README.